<launch>
    <!-- Load configuration parameters -->
    <rosparam file="$(find origami_sim)/config/soft_robot_sim_config.yaml" command="load"/>
    
    <!-- Launch FK -->
    <node name="soft_robot_sim" pkg="origami_sim" type="sim_FK.py" respawn="false" output="screen" />
    
    <!-- Launch vis -->
    <node name="soft_robot_sim_vis" pkg="origami_sim" type="origami_sim_vis.py" respawn="false" output="screen" />

    <!-- Launch low level control -->
    <node name="sim_controller" pkg="origami_sim" type="simulation_control_interface.py" respawn="false" output="screen" />

    <!-- Launch image view -->
    <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <param name="autosize" value="true" />
    <remap from="image" to="origami_vs/camera_image" />
    </node>
</launch>